### PR TITLE
fix for selinux issue-326 contiv/netplugin

### PR DESCRIPTION
**Pull request by gaurav-dalvi**
Fix for issue #326 under contiv/netplugin .
Need to permit Selinux for openvswitch_t profile so that Selinux can not block 6633 and 6634 ports.

### CODE REVIEW

File: roles/contiv_network/tasks/ovs_cleanup.yml [hunk 1]
```diff
@@ -25,6 +25,11 @@
 
 - debug: var=ports
 
+- name: permit openvswitch_t type in selinux
```
Comment (mapuri) [inline roles/contiv_network/tasks/ovs_cleanup.yml]: `permit` => `deny` or `disallow` ?

**Comment by mapuri** [comment]
LGTM other than one minor nit

**Comment by gaurav-dalvi** [comment]
Corrected it to : 
deny openvswitch_t type in selinux

**Comment by mapuri** [comment]
LGTM

**Pull request merged**